src/yki/registration/registration.clj

```diff
@@ -155,12 +154,16 @@
                                        :form_version   1
                                        :participant_id unified-participant-id}
               expiration-date         (registration->expiration-date registration-data)
+
+              payment-url              (or (:payment_url form) (get-payment-redirect-url payment-helper registration-id lang))
```

Comment:
Mikäs se IntellJ:n näppäinyhdistelmä olikaan koodin asettamiseen samalle tasolle riveittäin? Nyt `payment-url` ja `payment-link-expired-url`:ien arvo ei asetu samalle tasolle/sarakkeelle muiden arvojen kanssa. Ei tosin kovin kummoinen ongelma.

Reply:
Riippuu tietty konffeista, mulla taitaa olla `Cmd-Opt-L`

Reply:
Löytys, IntellJ:ssä asetus `Align let-binding values` Clojure Code Style alapuolella